back/app/services/sort_by_params_service.rb

```diff
@@ -85,7 +85,7 @@ def sort_activities(scope, params)
   def idea_voting_count_sort(scope, sort, phase_id, direction)
     if phase_id
       ids = IdeasPhase.where(phase_id: phase_id).order("#{sort} #{direction}").pluck(:idea_id)
-      Idea.unscoped.where(id: ids).order_as_specified(id: ids)
+      scope.order_as_specified(id: ids)
```

Comment:
Just respects what is passed in now - ie if the original query is for only published ideas then it will not return drafts